Add Chainstack 3.0 changes (#460)
chainstack/docs: docs/release-notes.md

# Release notes

## Chainstack 3.0

*June 30, 2022*

### What’s new

* **Protocols**.
  * Harmony migrated from testnet to [devnet](/operations/harmony/networks) for better experience.
  * Tezos migrated from Hangzhounet to [Ithacanet](/operations/tezos/networks).

> * **Pricing**. An updated pricing model with new features and increased included requests. See  <a href="https://chainstack.com/pricing-update" target="_blank">Introducing a new pricing plan, with MEV support, warp transactions, and affordable debugging and tracing API</a>.

Reviewer: Blog link should be updated

cc @py-goh

Reviewer: warp or Warp?

Reviewer: debugging and tracing or debug and trace?

Reviewer: Also extra whitespace `See  `

Reviewer: What is the new pricing plan being introduced?

Before:
- Developer
- Growth
- Business
- Enterprise

After:
- Developer
- Growth
- Business
- Enterprise

Maybe we should say that the plans were updated/refreshed or something?

Author: Fixed

> * **Pricing**. An update to the pricing plans with new features and increased included requests. See <a href="https://chainstack.com/pricing-update-june-2022" target="_blank">Introducing updated pricing plans, with MEV API, Warp transactions, and affordable debug and trace APIs</a>.

Reviewer: For consistency

```suggestion
* **Pricing**. An update to the pricing plans with new features and increased included requests. See <a href="https://chainstack.com/pricing-update-june-2022" target="_blank">Introducing updated pricing plans, with Warp transactions, MEV API, and debug and trace APIs on elastic Ethereum archive nodes</a>.
```

elastic Ethereum archive nodes?
Ethereum elastic archive nodes?

> * **Pricing**. An update to the pricing plans with new features and increased included requests. See <a href="https://chainstack.com/pricing-update-june-2022" target="_blank">Introducing updated pricing plans, with Warp transactions, MEV API, and debug and trace APIs on elastic Ethereum archive nodes</a>.

Reviewer: We might remove features here as it's not related to the pricing and we also talk about them below

```suggestion
* **Pricing**. An update to the pricing plans with new features and increased included requests. See <a href="https://chainstack.com/pricing-update-june-2022" target="_blank">Introducing updated pricing plans</a>.
```

* **Node naming**. Shared nodes are now [elastic nodes](/glossary/elastic-node) to reflect the underlying architecture and scalable infrastructure.

> * **API**.

Reviewer: ```suggestion
* **APIs**.
```
?

>   * [Debug and trace API](/operations/ethereum/debug-and-trace-api) for elastic Ethereum archive nodes.

Reviewer: Why is Warp not here?

Author: Added

>   * [Debug and trace API](/operations/ethereum/debug-and-trace-api) for elastic Ethereum archive nodes.

Reviewer: ```suggestion
  * [Debug and trace API](/operations/ethereum/debug-and-trace-apis) for elastic Ethereum archive nodes.
```
?

>   * [Debug and trace API](/operations/ethereum/debug-and-trace-apis) for elastic Ethereum archive nodes.

Reviewer: Probably worth reshuffling for consistency (i.e. Warp first)

Author: Done

  * [Miner Extractable Value (MEV) API](/operations/ethereum/mev-api) support for elastic Ethereum nodes for mainnet and Goerli testnet.

> * **Mempool**. You can now get fast transaction propagation with the **Warp transactions** feature on [Polygon](/operations/polygon/warp-transactions) and [Binance Smart Chain](/operations/bsc/warp-transactions).

Reviewer: Hmmm not sure why it deserves a separate category whereas it's just one more API / type of node

Author: Fixed
